src/config/index.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/no-anonymous-default-export */
```

Comment:
you will have other `env` variables, so it is better to create `const` for each and then export the desired config object.
you can create a `config.ts file` with the next code
` export const API_KEY = process.env.REACT_APP_API_KEY; .......`
create file `firebase.ts` for firebase config
`import {API_KEY} from './config'
export default {apiKey: API_KEY}
`
create `index.ts`
`export {default as firebaseConfig }from './firebase' ....`